examples/simple-rest/README_pt.adoc

```diff
@@ -0,0 +1,136 @@
+:index-group: REST
+:jbake-type: page
+:jbake-status: status=published
+
+= REST Simples
+
+Definir um serviço REST é fácil, simplesmente adicione a anotação ``@Path`` na classe, então defina o método HTTP (``@GET``, ``@POST``, ...) ao método da classe que deseja usar.
+
+= O código
+
+== O Serviço REST: ``@Path``, ``@GET``, ``@POST``
+
+Aqui nós temos um REST simples, anotamos a classe com ``@Path("/greeting")`` para indicar a rota correspondente a classe ``GreetingService``. Definimos ``message()`` como ``@GET`` e ``lowerCase()`` como ``@POST`` para esta rota ``/greeting``. Pronto, temos um serviço! Simples não?
+
+....
+package org.superbiz.rest;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+
+@Path("/greeting")
+public class GreetingService {
+    @GET
+    public String message() {
+        return "Hi REST!";
+    }
+
+    @POST
+    public String lowerCase(final String message) {
+        return "Hi REST!".toLowerCase();
+    }
+}
+....
+
+== Testando
+
+=== Teste para o serviço JAXRS
+
+Usamos o OpenEJB ``ApplicationComposer`` para facilitar o teste.
+
+A ideia é primeiro ativar os serviços ``jaxrs``. Isto é feito usando a anotação ``@EnableServices``.
+
+Então nós criamos a aplicação simplesmente retornando um objeto representando o ``web.xml``. Aqui nós simplesmente o usamos para definir o contexto raiz mas você também pode usar para definir sua aplicação REST também. E para completar a aplicação nos adicionamos a anotação ``@Classes`` para definir o conjunto de classes a ser utilizado nesse app.
```

Comment:
Hi Gabriel, thanks for contributing!
in this sentence: "E para completar a aplicação nos adicionamos a anotação" in the word "nos" you forgot the accent. Otherwise, your translation is great!

Reply:
Thank you for your revision @marcoantoniobferreira 
I added the accent :)